n_rrt_nodes: 100

min_bounds: [0, -10, 1]
max_bounds: [10, 10, 1.5]

near_radius_rrt: 1.5

delta_distance_rrt: 1.0

robot_radius: 0.5 

n_fail_iterations_rrt: 500

# arrays with size proportional to number of cameras
camera_info_p: 20000
camera_resolution: 10
max_depth: [8, 0.4] 

# frame ids
base_frame_id: base_frame
world_frame_id: world
cam_frame_id: [front_cam]

verbose: true
